string command_unit_list
---
string return_t   